plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -262,6 +262,17 @@ BOOL CALLBACK ConfigProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
             swprintf_s(temp, L"%d%%", configvol);
             SetWindowText(GetDlgItem(hWnd, IDC_VOLUME_LABEL), temp);
 
+            const int module = (int)SendMessage(GetDlgItem(hWnd, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
```

Comment:
Is an integer cast necessary here? (I don't think it's needed as ``CB_GETCURSEL`` returns the value of the index associated with the object, which already has an int data type)

Reply:
Actually cast to an enum class would be reasonable here. (But also using the same enum class as datatype for module).